docs/getting-started.md

```diff
@@ -45,36 +46,50 @@
 [brew]: https://brew.sh/
 [go]: https://golang.org/dl/
 [jq]: https://stedolan.github.io/jq/download/
-[kind]: https://sigs.k8s.io/kind
-[kubectl]: https://kubernetes.io/docs/tasks/tools/install-kubectl/
```

Comment:
why remove kubectl?

Reply:
I think I added it above as "install the Kubernetes CLI"